buildSrc/src/main/kotlin/Config.kt

```diff
@@ -4,7 +4,7 @@ object Config {
     const val minSdk = 21
 
     const val major = 0
-    const val minor = 10
+    const val minor = 11
     const val patch = 1
```

Comment:
Should be then 0.11.0 no?

Reply:
true. I updated it to do a feature release.  I wanted to rollback it but kept the 0.11.1. Didn't wanted to change it.